README.md

```diff
@@ -17,30 +18,10 @@ cd Week\ 00/SLU00\ -\ Jupyter\ Notebook/
 pip freeze > requirements.txt
 ```
 
-To enable students to submit a LU you'll need to add 3 cells at the bottom of the notebook, as follows:
+4. To enable students to submit a SLU, please copy the cells regarding work submission on `instructores_example/Exercise Notebook.ipynb`, to yours.
 
 <img src='assets/submit.png' alt='Finder' width="75%" />
 
-**Cell 1** markdown, read-only
```

Comment:
despite this info also being in the in the example notebook, i want to keep it here because instructors will quickly be able to know exactly what to put in their exercise notebooks based on this info